tests/virt_autotest/kubevirt_tests_server.pm

```diff
@@ -181,13 +208,26 @@ sub install_kubevirt_pkgs_n_deploy_manifests {
     if ($virt_manifests_repo) {
         zypper_call("in -f -r Virt-Manifests-Repo $virt_manifests");
     } elsif (script_run("rpmquery $virt_manifests")) {
-        zypper_call("in -f -r SLE-Module-Containers${os_version}-Updates $virt_manifests");
+        if (is_sle_micro) {
+            zypper_call("in -f $virt_manifests");
+        } else {
+            zypper_call("in -f -r SLE-Module-Containers${os_version}-Updates $virt_manifests");
```

Comment:
What if SLE-Module-Containers${os_version}-Updates does not have update for manifest. eg sle15sp5? Zypper will automatically install the highest version, right? Why is here needed to specify repo?

Reply:
The default repo is from the build installer, and the variable VIRT_MANIFESTS_REPO is for testing the kubevirt packages from devel repo or others. We need to test kubevirt packages along with the build.

Reply:
I know the story that during 15sp4, it is needed to install the pkg from SLE-Module-Containers${os_version}-Updates, but that's an exception, and it won't always be the case.

Reply:
Yes, it works well in 15sp5 test. As we discussed offline, will add a judgment to check if manifests packages are in the default repo or not.